src/customize-pop/gnome-tweaks-extensions/user-themes.md

```diff
@@ -0,0 +1,94 @@
+# Managing User Themes
+
+User themes include changes to window animations, desktop color schemes, window designs, icons, and workspaces. You can install custom GNOME Themes from opendesktop.org. The Pop!\_OS default themes can be re-applied using GNOME Tweaks or the by using Terminal commands.
+
+>**Note**: Applying custom GNOME theming options may cause system instability or glitchy behavior. Additionally, custom themes are not subject to testing by the System76 Quality Assurance team or application developers.
+
+## Enabling User Themes
+
+1. Launch the GNOME Extension Manager.
```

Comment:
You're going to need to link back to the previous page for this, because this third-party app is not installed by default. (We install the `chrome-gnome-shell` support package for [extensions.gnome.org](https://extensions.gnome.org/) by default, although the browser extension still needs to be manually installed.)